db/seeds.rb

```diff
@@ -24,7 +24,14 @@
 # Configs für den Abfallkalender
 StaticContent.create(name: "wasteTypes", data_type: "json", content: File.read("doc/waste_types.json") )
 
-def create_web_url = File.read("doc/waste_types.json")
+# For Noticeboard Mailer (Preview)
+StaticContent.create(
+  name: "noticeboard-notify-creator",
+  data_type: "json",
+  content: "{\"app_name\":\"Smart Village App\",\"mail_footer\":[\"Fußzeile Zeile 1\",\"Fußzeile Zeile 2\"]}"
```

Comment:
i found a different content here as in the code existing:

```
content: "{\"app_name\":\"Test-App\",\"mail_footer\":[\"Zeile 1\", \"\", \"Zeile 2\"]}"
```

what was the reason to chose a different content here? i giess having the same will result in less questions in the future (and would have saved my comment question comment now ;))

Reply:
content: "{\"app_name\":\"Test-App\",\"mail_footer\":[\"Zeile 1\", \"\", \"Zeile 2\"]}" 
Here its using the first_or_create method, and there's already a StaticContent object within the seeds. (but with that StaticContent it does not work).

Reply:
then i don't understand the reason, why we create the content in seeds? in the code we have `first_or_create` to ensure that object in the database. how come that we need it additionally for the preview? and if we need it for the preview in the seeds, why can't that be the same content?

Reply:
app/mailers/noticeboard_mailer.rb
```
    def set_static_content_values
      static_content = StaticContent.find_by(name: "noticeboard-notify-creator").try(:content)

      begin
        static_content = JSON.parse(static_content)
      rescue StandardError
        return
      end

      @app_name = static_content["app_name"]
      @mail_footer = static_content["mail_footer"].join("\n")
    end
```
maybe change into `find_or_create_by` instead of `first_or_create` ?

Reply:
aah. now i understand. i hoped, that `first_or_create` with some params will return `first` considering that params and not the very first first. so it seems that `find_or_create_by` does exactly what the code should do already. you can change that, yes. and please have a check for other `first_or_create`s in the code, that could be bad as well. thanks.

Reply:
We acould also use something like this:
`StaticContent.where(something: value).first_or_create(attributes)`
?
I made the change only in the seeds for testing purposes to preview the mailer. Maybe make a new PR with  `where.first_or_create` or `find_or_create_by` ?

Reply:
Yes, you are right. However, switching to `where.first_or_create `or `find_or_create_by` could potentially impact the production database.

Reply:
So let's do it n this PR to not add the creation here and remove it in another PR.

Reply:
Let's do this in a separate PR to avoid bad side effects.

Reply:
Okay, I removed the second commit